### PR TITLE
Changelog for 0.26.0

### PR DESCRIPTION
**Pull request by clalancette**
Once this is approved, I'll merge it, tag it, and then bloom-release it into `rolling`.  @MichaelOrlov FYI

### CODE REVIEW

File: ros2bag/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Changelog for package ros2bag
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Add option to disable recorder keyboard controls (`#1607 <https://github.com/ros2/rosbag2/issues/1607>`_)
+* Support service 2/2 --- rosbag2 service play (`#1481 <https://github.com/ros2/rosbag2/issues/1481>`_)
+* Added exclude-topic-types to record (`#1582 <https://github.com/ros2/rosbag2/issues/1582>`_)
+* Contributors: Alejandro Hernández Cordero, Barry Xu, Bernd Pfrommer
```
Comment (MichaelOrlov) [inline ros2bag/CHANGELOG.rst]: Please add me as a contributor as well. I contributed in all 3 PRs as co-author.
```suggestion
* Contributors: Alejandro Hernández Cordero, Barry Xu, Bernd Pfrommer, Michael Orlov
```

File: rosbag2_cpp/CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,13 @@
 Changelog for package rosbag2
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Support service 2/2 --- rosbag2 service play (`#1481 <https://github.com/ros2/rosbag2/issues/1481>`_)
+* Use middleware send and receive timestamps from message_info during recording (`#1531 <https://github.com/ros2/rosbag2/issues/1531>`_)
+* Update to use yaml-cpp version 0.8.0. (`#1605 <https://github.com/ros2/rosbag2/issues/1605>`_)
+* Contributors: Barry Xu, Chris Lalancette, jmachowinski
```
Comment (MichaelOrlov) [inline rosbag2_cpp/CHANGELOG.rst]:
```suggestion
* Contributors: Barry Xu, Chris Lalancette, jmachowinski, Michael Orlov
```

File: rosbag2_storage_mcap/CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,14 @@
 Changelog for package rosbag2_storage_mcap
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Support service 2/2 --- rosbag2 service play (`#1481 <https://github.com/ros2/rosbag2/issues/1481>`_)
+* Use middleware send and receive timestamps from message_info during recording (`#1531 <https://github.com/ros2/rosbag2/issues/1531>`_)
+* Update to use yaml-cpp version 0.8.0. (`#1605 <https://github.com/ros2/rosbag2/issues/1605>`_)
+* Check existence of a file before passing it to the mcap reader (`#1594 <https://github.com/ros2/rosbag2/issues/1594>`_)
+* Contributors: Barry Xu, Chris Lalancette, Christopher Wecht, jmachowinski
```
Comment (MichaelOrlov) [inline rosbag2_storage_mcap/CHANGELOG.rst]:
```suggestion
* Contributors: Barry Xu, Chris Lalancette, Christopher Wecht, jmachowinski, Michael Orlov
```

File: rosbag2_storage_sqlite3/CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,13 @@
 Changelog for package rosbag2_storage_default_plugins
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Forthcoming
+-----------
+* Support service 2/2 --- rosbag2 service play (`#1481 <https://github.com/ros2/rosbag2/issues/1481>`_)
+* Use middleware send and receive timestamps from message_info during recording (`#1531 <https://github.com/ros2/rosbag2/issues/1531>`_)
+* Update to use yaml-cpp version 0.8.0. (`#1605 <https://github.com/ros2/rosbag2/issues/1605>`_)
+* Contributors: Barry Xu, Chris Lalancette, jmachowinski
```
Comment (MichaelOrlov) [inline rosbag2_storage_sqlite3/CHANGELOG.rst]:
```suggestion
* Contributors: Barry Xu, Chris Lalancette, jmachowinski, Michael Orlov
```

**Review by MichaelOrlov — CHANGES_REQUESTED** [review]
A few nitpicks to add me as a contributor to some packages as I was a co-author for many PRs which are metioned there.

**Comment by clalancette** [comment]
> A few nitpicks to add me as a contributor to some packages as I was a co-author for many PRs which are metioned there.

Yeah, I was noticing that.  As it stands, `catkin_generate_changelog` only looks at the author of the commit; it doesn't look at any `Signed-off-by` or `Co-authored` lines.  We probably should change that.

**Review by MichaelOrlov — APPROVED** [review]
LGTM

**Comment by clalancette** [comment]
The PR job failed, but the rest were successful.  I'm going to go ahead and merge this one, then do the release.

**Pull request merged**